### DISCUSSION TITLE
Integration broken on core update to 2023.9.3

### DISCUSSION THREAD

**Discussion by yebaws** [BottlecapDave/HomeAssistant-OctopusEnergy#436] [General]
I did a core update from 2023.8.3 to 2023.9.3 and started getting "This entity is no longer being provided by the octopus_energy integration. If the entity is no longer in use, delete it in settings." for all of my entities.

Rolled back using the last full back up to 2023.8.3 and the problem resolved itself.

I've been through this process twice with the same results.

I need this great integration for so many of my automations, but don't want to be stuck on the same core version forever.

Any ideas please?

**Comment by BottlecapDave** [comment]
Hello. I'm currently successfully using the integration on the specified version. Are you able to follow the [faq](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/faq.md#ive-been-asked-for-my-meter-information-in-a-bug-request-how-do-i-obtain-this) and paste the results here? It would also be useful if you're able to paste the logs from your HA instance.

**Reply by yebaws** [reply]
I updated OctopusEnergy this morning and then tried the core update again. It's all working now! Updating the integration was my first thought previously, but I hadn't realised that I needed to go into the HACS menus to find the version / update notification - had thought that it would appear with other updates in the system menu.
Thanks very much for following up anyway.

**Reply by BottlecapDave** [reply]
I suspected that might have been the issue as a few other people have fallen over that hurdle. It would be nice if HACS pushed the update to the main update section. The HACS integration provides a sensor to indicate pending updates, which I have an automation for to notify me through other means.

**Comment by yebaws** [comment]
> which I have an automation for to notify me through other means.
Thanks for the tip. I see the sensor now. I'll have a play with it....anything beats doing some work
